aws_eden_core/methods.py

```diff
@@ -278,7 +267,7 @@ def update_service(reference_service: dict, resource_name: str, task_definition_
     return response
 
 
-def config_add_env(bucket_name: str, key: str, env_name: str, env_cname: str, env_type: str, update_key: str):
+def endpoints_add(bucket_name: str, key: str, env_name: str, env_cname: str, env_type: str, update_key: str):
```

Comment:
Please keep consistency with the naming of function arguments used in `endpoints_delete`. 

```suggestion
def endpoints_add(bucket_name: str, key: str, name: str, fqdn: str, env_type: str, update_key: str):
```